<?xml version="1.0"?>
<launch>
	<rosparam command="load" file="$(find structure_core)/launch/structure_prameters.yaml"/>
	 <node pkg="structure_core" type="structure_driver" name="structure_driver" output="screen"/>
<node pkg ="tf" type="static_transform_publisher" name="pcl2baselink" args="0 0 0 0 0 0 base_link camera_depth_optical_frame 100" />
 	 <node pkg="nodelet" type="nodelet" args="manager" name="record_player_manager" output="screen"/>

<node pkg="nodelet" type="nodelet" name="convert" args="load depth_image_proc/convert_metric record_player_manager --no-bond" >
	<remap from="image_raw" to="/depth/image"/>
	<remap from="image" to="/depth/image_f32"/>

</node>
/*Remapping for rtabmap_ros -> rtab.launch >> make configurable*/
<remap from="/camera/depth_registered/image_raw" to="/depth/image"/>

<node pkg="nodelet" type="nodelet" name="cloudify_front" args="load depth_image_proc/point_cloud_xyz record_player_manager --no-bond" >
	<remap from="image_rect" to="/depth/image_f32"/>
	<remap from="camera_info" to="/depth/camera_info"/>
</node>



</launch>
